src/Bundle/Maker/MakeFactoryPHPDocMethod.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Zenstruck\Foundry\Bundle\Maker;
+
+final class MakeFactoryPHPDocMethod
```

Comment:
this is just an experiment, I wanted to see where it goes with a VO for that. I think I prefer this instead of the complex templating stuff. WDYT?